contexts/chat-context.provider.tsx

```
@@ -100,30 +88,31 @@ export const ChatContextProvider: React.FC<PropsWithChildren> = ({
       const data = await response.json();
 
       if (!!data.token) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)